core/jazz_acl/swagger/swagger.json

```diff
@@ -0,0 +1,1266 @@
+                "application/json": "$input.json('$')"
+              },
+              "responseParameters": {
+                "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+                "method.response.header.Access-Control-Allow-Methods": "'*'",
+                "method.response.header.Access-Control-Allow-Origin": "'*'"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Default response for CORS method",
+            "headers": {
+              "Access-Control-Allow-Headers": {
+                "type": "string"
+              },
+              "Access-Control-Allow-Methods": {
+                "type": "string"
+              },
+              "Access-Control-Allow-Origin": {
+                "type": "string"
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/400ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "$ref": "#/definitions/401ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "Forbidden",
+            "schema": {
+              "$ref": "#/definitions/403ErrorResponse"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "$ref": "#/definitions/404ErrorResponse"
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "$ref": "#/definitions/500ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "acl"
+        ],
+        "description": "GET method to get the permission for a specific service for a user",
+        "operationId": "getPermission",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [{
+          "name": "userId",
+          "in": "query",
+          "description": "userId of the user",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "name": "serviceId",
+          "in": "query",
+          "description": "service Id",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "name": "permission",
+          "in": "query",
+          "description": "permission",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "name": "category",
+          "in": "query",
+          "description": "service category",
+          "required": true,
+          "type": "string"
+        }],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/200SuccessResponse"
```

Comment:
This can be a simple object - {"authorized": true/false}?

Reply:
using a common success response